src/pages/settings/PasswordPage.js

```diff
@@ -71,6 +71,8 @@ class PasswordPage extends Component {
 
         if (this.state.newPassword && this.state.confirmNewPassword && !this.doPasswordsMatch()) {
             stateToUpdate.shouldShowPasswordConfirmError = true;
+        } else {
+            stateToUpdate.shouldShowPasswordConfirmError = false;
         }
 
         if (!isEmpty(stateToUpdate)) {
```

Comment:
Just realized this, but this check does nothing, since we are setting variables in the `else` blocks.

Reply:
Well, if I enter mismatch passwords first and then I update the `newPassword` to match the `confirmPassword`, the `onBlureNewPassword` still showed the `Password must match` error. Hence, I added this check.

Reply:
Okay. Got it. Will update. I thought it was about the `else` block.

Reply:
Updated